### DISCUSSION TITLE
FireworkStar Color

### DISCUSSION THREAD

**Discussion by Ativendus** [Minestom/Minestom#1525] [Help] [answered]
How do I change the color of the item itself? Here is my current code and my expectation. 

```kotlin
inventory.setItemStack(0, ItemStack.of(Material.FIREWORK_STAR).withMeta(FireworkMeta::class.java) { meta ->
    meta.effects(listOf(FireworkEffect(true, true, FireworkEffectType.SMALL_BALL, listOf(Color(0, 255, 30)), listOf(Color(0, 255, 30)))))
})
```

Expected...
![Screenshot 2022-11-17 180700](https://user-images.githubusercontent.com/66266113/202511380-3722bde5-73fa-4a59-abdc-f4eac0ea2999.png)

Minestom...
![Screenshot 2022-11-17 180728](https://user-images.githubusercontent.com/66266113/202511383-77f9498a-b89d-4947-9dde-49ad7176ef3b.png)

**Comment by emortaldev** [comment, marked as answer]
```kt
inventory.setItemStack(0, ItemStack.of(Material.FIREWORK_STAR).withMeta(FireworkEffectMeta::class.java) { meta ->
    meta.effect(FireworkEffect(true, true, FireworkEffectType.SMALL_BALL, listOf(Color(0, 255, 30)), listOf(Color(0, 255, 30))))
})
```
Turns out you just needed to use the FireworkEffectMeta instead.
